sounds/templatetags/display_sound.py

```diff
@@ -19,63 +19,114 @@
 #
 
 from __future__ import absolute_import
-from sounds.models import Sound
+
 from django import template
-from django.conf import settings
+
+from sounds.models import Sound
 
 register = template.Library()
 
 
 @register.inclusion_tag('sounds/display_sound.html', takes_context=True)
-def display_sound(context, sound):
-    """
-    When a sound object is passed make sure to call select_related for license and user so it's already fetched
+def display_sound(context, sound, player_size='small'):
+    """This templatetag is used to display a sound with its player. It prepares some variables that are then passed
+    to the display_sound.html template to show sound information together with the player. This function is supposed
+    to be called from Django templates so it will render the display_sound.html template with the computed variables
+    and then add it to the tempalte from which it was called.
+
+    Args:
+        context (django.template.Context): an object with contextual information for rendering a template. This
+          argument is automatically added by Django when calling the templatetag inside a template.
+        sound (int or Sound): sound ID or Sound object of the sound that will be shown. If no sound exists for the
+          given ID, the display_sound.html will be rendered with empty HTML.
+        player_size (str, optional): size of the player to display. Must be one of ['small', 'big_no_info',
```

Comment:
perhaps some detail about what the _no_info variants mean? Is there a 'big' as well?

Reply:
it is not yet clear how many there will be. but yeah, we need details for all of them. I'll make sure we have this.